media/icon/light/refresh.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I wonder the license of this file.
How about mentioning the URL(?) where you get this file as comment inside this file?

Reply:
/cc @llFreetimell

Reply:
Good point, if images are not created by ourselves, copyright and license should be checked.

Reply:
This icon is came from [vscode-icon](https://github.com/microsoft/vscode-icons). And there is no license issue for using this icon. I updated `LICENSE` file. Thank you.

Reply:
My intention was inserting like following :D

```suggestion
<!--
This image is copied from
https://github.com/microsoft/vscode-icons/blob/afa041d46f4323acdad167afdded418db08cd1da/icons/light/refresh.svg
-->
<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Reply:
Should I use blob address? Can I use https://raw.githubusercontent.com/microsoft/vscode-icons/master/icons/light/refresh.svg address?

Reply:
I am not sure but it is better to write trackable URL :)
IMHO, the raw address is not sure for tracking...!

Reply:
_master_ could be changed to different licensing policy later. IMHO, URL with commit hash would be great.